src/Pagination/Pagination.Component.js

```diff
@@ -45,6 +45,21 @@ export const PaginationComponent = () => {
                         description:
                             'number (default: page 1) - Initial page to be selected.'
                     },
+                    {
+                        name: 'displayTotalProps',
+                        description:
+                            'object - additional props to be spread to the display total section'
+                    },
+                    {
+                        name: 'linkProps',
```

Comment:
I missed the original PR (#224), but should we have two separate props for the "previous" and "next" links?

Reply:
Separate props as in a prevNextProps which is different than the linkProps or as in prevProps and nextProps

Reply:
I was asking about `prevProps` and `nextProps`.

Reply:
ok, i just added a prevProps and nextProps values to the pagination control